src/main/java/uk/ac/ebi/subs/fileupload/config/MessagingConfiguration.java

```diff
@@ -26,6 +27,10 @@
     public static final String USI_FILE_DELETION_QUEUE = "usi-file-deletion";
     private static final String EVENT_FILE_DELETION = "usi.file.deletion";
 
+    public static final String GLOBUS_SHARE_REQUEST_QUEUE = "usi-fu-globus-share-request";
```

Comment:
Good question. I actually put it here because file-upload-service is the only one that is going to create a queue with this name and make any use of it. I think you re confusing routing keys with queue names here. subs-messaging only contains routing keys as they are needed in two services i.e. subs-api and file-upload-service.